_includes/header.html

```diff
@@ -16,24 +16,24 @@
                     <li>
                         <a href="/" title="There's no place like Home...">Home</a>
                     </li>
+                    <li>
+                        <a href="/docs/getting-started" title="Our Getting Started guide">Get Started</a>
+                    </li>
                     <li class="dropdown">
-                        <a class="dropdown-toggle" href="#" data-toggle="dropdown" title="Documentation on using ev3dev">
-                            Docs
+                        <a class="dropdown-toggle" data-toggle="dropdown" title="Documentation on using ev3dev">
+                            Documentation
                             <span class="caret"></span>
                         </a>
                         <ul class="dropdown-menu">
                             <li>
-                                <a href="/docs/getting-started">Getting Started</a>
-                            </li>
-                            <li>
                                 <a href="/docs/tutorials">Tutorials</a>
                             </li>
                             <li role="separator" class="divider"></li>
                             <li>
-                                <a href="/docs/motors">Motors</a>
+                                <a href="/docs/libraries">Libraries</a>
                             </li>
                             <li>
-                                <a href="/docs/sensors">Sensors</a>
+                                <a href="/docs/driver-access">Low-level driver access</a>
```

Comment:
Yeah, I can't remember at this point either, but I think we do have a comment for this somewhere else. I'll change this to match whatever we do over there.